Patch 1.0 RTL with Lint Fixes

calebofearth opened the pull request:
@nstewart-amd
@amullick007

amullick007 commented:
> @nstewart-amd @amullick007 Couldn't add you as reviewers so tagging here :)

Do you want me to run Lint and RDC on this PR?

calebofearth commented:
> > @nstewart-amd @amullick007 Couldn't add you as reviewers so tagging here :)
> 
> Do you want me to run Lint and RDC on this PR?

Yes, I think there are some slight changes since last time you ran on this branch.

amullick007 commented:
> > > @nstewart-amd @amullick007 Couldn't add you as reviewers so tagging here :)
> > 
> > 
> > Do you want me to run Lint and RDC on this PR?
> 
> Yes, thanks, I think there are some slight changes since last time you ran on this branch. EDIT: I need to update the HW_REV_ID first.

Is this branch different from what we have at TOT? I mean does this have the 1.1 changes as well? Or this is just 1.0 + Lint fixes?

calebofearth commented:
> > > > @nstewart-amd @amullick007 Couldn't add you as reviewers so tagging here :)
> > > 
> > > 
> > > Do you want me to run Lint and RDC on this PR?
> > 
> > 
> > Yes, thanks, I think there are some slight changes since last time you ran on this branch. EDIT: I need to update the HW_REV_ID first.
> 
> Is this branch different from what we have at TOT? I mean does this have the 1.1 changes as well? Or this is just 1.0 + Lint fixes?

Only 1.0 code + lint fixes. I just pushed updates to HW_REV_ID, as well as README+Release_Notes. The branch is good to run your checks now. Thanks!

amullick007 commented:
> > > > > @nstewart-amd @amullick007 Couldn't add you as reviewers so tagging here :)
> > > > 
> > > > 
> > > > Do you want me to run Lint and RDC on this PR?
> > > 
> > > 
> > > Yes, thanks, I think there are some slight changes since last time you ran on this branch. EDIT: I need to update the HW_REV_ID first.
> > 
> > 
> > Is this branch different from what we have at TOT? I mean does this have the 1.1 changes as well? Or this is just 1.0 + Lint fixes?
> 
> Only 1.0 code + lint fixes. I just pushed updates to HW_REV_ID, as well as README+Release_Notes. The branch is good to run your checks now. Thanks!

RDC is clean on this PR

calebofearth commented:
@mkurc-ant Can you take a look at the workflow issue to see if there's an easy fix? Something new going on with SCCACHE.

tmichalak commented:
> @mkurc-ant Can you take a look at the workflow issue to see if there's an easy fix? Something new going on with SCCACHE. EDIT: I just updated from version 0.3.3 to 0.8.1 and it seems OK now.

@calebofearth I guess once this is merged you will also merge https://github.com/chipsalliance/caliptra-rtl/pull/559. Unless you want to cherry-pick the fixes from #559 here.

calebofearth commented:
> > @mkurc-ant Can you take a look at the workflow issue to see if there's an easy fix? Something new going on with SCCACHE. EDIT: I just updated from version 0.3.3 to 0.8.1 and it seems OK now.
> 
> @calebofearth I guess once this is merged you will also merge #559. Unless you want to cherry-pick the fixes from #559 here.

@tmichalak Thanks for the suggestion. I cherry-picked those updates and reverted the SCCACHE version change.
